test/distribution/test_inflated_beta.py

```diff
@@ -0,0 +1,50 @@
+import mxnet as mx
+import pandas as pd
+
+from gluonts.dataset import common
+from gluonts.model import deepar
+from gluonts.mx.distribution.inflated_beta import ZeroAndOneInflatedBetaOutput
+from gluonts.mx.trainer import Trainer
+
+
+def test_symbol_and_array():
+    # Tests for cases like the one presented in issue 1211, in which the Inflated
+    # Beta outputs used a method only available to arrays and not to symbols.
+    # We simply go through a short training to ensure no exceptions are raised.
+    data = [
+        {
+            "target": [0, 0.0460043, 0.263906, 0.4103112, 1],
+            "start": pd.to_datetime("1999-01-04"),
+        },
+        {
+            "target": [1, 0.65815564, 0.44982578, 0.58875054, 0],
+            "start": pd.to_datetime("1999-01-04"),
+        },
+    ]
+    data = common.ListDataset(data, freq="W-MON", one_dim_target=True)
+
+    trainer_hybrid = Trainer(ctx=mx.context.gpu(), epochs=1, hybridize=True)
+
+    trainer_not_hybrid = Trainer(
+        ctx=mx.context.gpu(), epochs=1, hybridize=False
```

Comment:
I would set the context to `cpu` here (or better leave it to default) since that's the only one available on the instances running the tests on GitHub I think.

You can also set `num_batches_per_epoch=2` to make the test very fast. I think the point here is to just to have a smoke test (just check that it runs).